homework/style.css

```diff
@@ -1,3 +1,35 @@
 .alert-error {
-  color: red;
+  color: rgb(128, 50, 50);
+  background-color: rgb(248, 215, 218);
+  padding: 10px 10px;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+.repos {
+  list-style: none;
+  border: 2px solid rgb(229, 229, 229);
+  padding: 20px 10px;
+}
+
+.header {
+  background-color: rgb(63, 81, 181);
+  border: 2px solid rgb(229, 229, 229);
+  padding: 20px 10px;
+  color: white;
+}
+
+#root {
+  max-width: 800px;
```

Comment:
Your app would look better if the content was centered horizontally. You only need to add this to `#root`:

```css
margin: 0 auto;
```